lib/oli_web/templates/layout/chromeless.html.eex

```diff
@@ -46,7 +46,10 @@
     <!-- Janus Part Component external CSS Files -->
     <link rel="stylesheet" href="/css/janus_image_carousel_delivery.css"/>
 
-    <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
+<% Map.get(assigns, :vr_agent_active) |> IO.inspect(label: "---A1") %>
```

Comment:
Remove this line as it seems like it is here just for debugging

Reply:
Good catch! Thanks.